.nuspec/Xamarin.Forms.SingleProject.props

```diff
@@ -0,0 +1,42 @@
+<!-- "Single Project" support - all <ItemGroup/>'s must check if $(EnableDefaultItems) and $(SingleProject) are true -->
+<Project>
```

Comment:
Do I need to name this `Microsoft.Maui.SingleProject.props/targets`?

Reply:
Microsoft.Maui.Controls.SingleProject

Reply:
This is the similar to `Xamarin.Forms.props`, does it really need `.Controls` in the name?

Reply:
Ok, I guess it is: https://github.com/xamarin/Xamarin.Forms/tree/main-handler/.nuspec